server/services/licenceService.ts

```diff
@@ -583,6 +583,18 @@ export default class LicenceService {
     return this.licenceApiClient.getLicenceById(licence.variationOf.toString(), user)
   }
 
+  async getLicenceVariations(nomisId: string): Promise<LicenceSummary[]> {
```

Comment:
I think this name could be easily misconstrued to be 'get all the licences that are variations of the given licence' (ie including those that are active).
Perhaps a more fitting name would be something like `getIncompleteLicenceVariations`? I'm hesitant to say 'in progress' or 'non-active' as those are licence statuses in themselves, but I think 'incomplete' covered both not active and not inactive.

Reply:
good point, of course other status can be variations of a licence. Updated.